modules/node_modules/@ncigdc/components/QuickSearch/QuickSearchResults.js

```diff
@@ -149,7 +147,9 @@ export default ({ results = [], query, isLoading, onSelectItem, onActivateItem }
           onClick={() => onActivateItem(item)}
         >
           <div style={styles.itemIconWrapper}>
-            <ResultIcon type={item._type} style={styles.itemIcon} />
+            <span style={styles.itemIcon}>
+              {getIconClassNameByEntityType(item._type)}
```

Comment:
`getIconClassNameByEntityType` this function should probably be renamed as it's no longer returning a class name